guides/advanced_guides/field_properties.md

```diff
@@ -0,0 +1,36 @@
+# Field properties
+
+Every document field can have either or both or none of the following properties that can be modified in the [settings](/references/settings.md):
```

Comment:
Could be interesting to say again that they are both by default.
```suggestion
Every field in a document is by default searchable and displayed. These properties can be modified in the [settings](/references/settings.md):
```

Reply:
Yes, it would be much more clear!